src/crons/XKCD.ts

```diff
@@ -0,0 +1,82 @@
+import { Cron, findTextChannelByName } from '../framework';
+import got from 'got';
+import { parse } from 'node-html-parser';
+import { decode } from 'html-entities';
+import { KeyValue } from '../database';
+import { EmbedBuilder } from 'discord.js';
+
+const DATETIME_FMT: Intl.DateTimeFormatOptions = {
```

Comment:
le linter dit `'DATETIME_FMT' is assigned a value but never used`

Reply:
ah bah oui, vu que je passe la date directement au champs timestamp, plus besoin de formater la tade